### DISCUSSION TITLE
Editable mail templates

### DISCUSSION THREAD

**Discussion by Poseidon281** [pelican-dev/panel#370] [Ideas]
For example using the already installed [Monaco Editor](https://filamentphp.com/plugins/abdelhamid-errahmouni-monaco-editor#usage). I would like it if the mail templates can be edited.

**Comment by felinusfish** [comment]
Would like to mention that I've had Google flag my emails as spam on occasion, this is the only service that I've had that happen so far, which is a little frustrating but ... not much that can be done unless I can make the emails look better so that I can pass the Google spam filter far more effectively.
